projects/karchive/build.sh

```diff
@@ -0,0 +1,67 @@
+# Build zlib
```

Comment:
Could you please add the copyright to this file as well? It should've been auto-created by `infra/helper.py` script, or you can just copy that from any other `build.sh`.

Reply:
he he, the one i copied it from had no header, added.

Reply:
hm, I still don't see the copyright :) Did you push the change?